<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>457</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>505</width>
    <height>457</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Registro Alumnos [Base de Datos]</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../OneDrive - Universidad Autonoma de Nuevo León/FACPYA/Primer Semestre/iconos/facpya logo.ico</normaloff>../../../OneDrive - Universidad Autonoma de Nuevo León/FACPYA/Primer Semestre/iconos/facpya logo.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>196</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_Nombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PRIMER NOMBRE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apppat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>APP PATERNO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_appmat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>77</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>APP MATERNO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Matricula">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>7</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MATRICULA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_carrera">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>L.T.I</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L.A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C.P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L.N.I</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_carrera">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CARRERA:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Tabla_Datos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>321</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>Matricula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A_Paterno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A_Materno</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Domicilio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ciudad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Carrera</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Beca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Foraneo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ingles</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prog</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contab</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estadistica</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Inv_Op</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_Registrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Limpiar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Grabar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar a DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Leer">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Eliminar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_nombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>28</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_apppat">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>51</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_matricula">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>4</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_appmat">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>75</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Consultar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_prog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Programación</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_bd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Base de Datos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_contab">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Contabilidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_edad">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_estadistica">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Estadística</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_inv_op">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Investigación de Operaciones</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_ciudad">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>San Nicolás</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apodaca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monclova</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saltillo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tampico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reynosa</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_ciudad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ciudad</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_estado">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>173</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_estado">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>170</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tamaulipas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nuevo León</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coahuila</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="chk_foraneo">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>170</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Foraneo</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_ingles">
    <property name="geometry">
     <rect>
      <x>416</x>
      <y>170</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Habla Inglés</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_domicilio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>102</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DOMICILIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_domicilio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_genero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Genero</string>
    </property>
    <widget class="QRadioButton" name="radio_hombre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hombre</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mujer">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>16</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mujer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_beca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Porcentaje de Beca</string>
    </property>
    <widget class="QRadioButton" name="radio_cero">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>42</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0 %</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_50">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_80">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>52</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>80%</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_100">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>100 %</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_MatplotLib">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MatPlotLib</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Seaborn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Seaborn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Connect_DB">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar DB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_matricula_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Actualizar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Imprimir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimir</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
